server/src/lib/lsp/completion/minecraft/json/document.ts

```diff
@@ -84,6 +85,8 @@ function performJsonCompletion(context: JsonCompletionContext) {
 
   onCompletionJsonMolang(context);
 
+  if (!Manager.Settings.Completion.JSON) return;
```

Comment:
Think you want this up top, it will already have done molang completion 🤔

Reply:
I want it to include the molang completion, the setting is for dynamic completion only, albeit it does suggest the molang at the wrong places.